modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -12,19 +12,17 @@ Runtime templates related to Anypoint Partner Manager 2.5 are upgraded when:
 
 == Perform the Upgrade
```

Comment:
As this is an imperative title, I expected steps. 
Not convinced that we need a subtopic here. 
How about going straight into 
"When a new template is available, the blah button in this location is activated. Before you click it, know that this can take some time and that it is safest for you to first upgrade in a nonproduction environment and test the new template before activating it in production."
The narrative is a little choppy as-is.

Reply:
Proofread below